security/azure/resources_with_public_ip_address/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v1.0
```

Comment:
version should be v2.0

Reply:
fixed this.

Reply:
Hey Curt, The purpose of this policy to be part of the security discovery process of the cloud and start the be the conversation starter among the customer's cloud team so that they can identify if these resources should really have public IP address.  Also, I am very new to contributing to the policy template repo and the process around, hence I apologize in advance if things are missing here.

Reply:
Added root readme.